src/org/sosy_lab/java_smt/api/QuantifiedFormulaManager.java

```diff
@@ -72,4 +84,6 @@ BooleanFormula mkQuantifier(
    */
   BooleanFormula eliminateQuantifiers(BooleanFormula pF)
       throws InterruptedException, SolverException;
+
+  void setOptions(ProverOptions... opt);
```

Comment:
This method does not make sense. A `ProverOption` is clearly meant for the prover.
Also, why not just use the integrated configuration framework for options and hand it to the manager from the context?

Reply:
Handled in newest commits. Thanks!